pylexirumah/plot_parameters.py

```diff
@@ -1,79 +1,24 @@
+# coding: utf8
 """Plot the number of filled-in parameters for each language.
 
 parameter_sampled: map languages to sets of parameters
 """
-
-# CLI libraries
```

Comment:
Follow [pep8](https://www.python.org/dev/peps/pep-0008/#imports) for order of imports.